### DISCUSSION TITLE
[CLIENT-ADDON] I want to know how to release a patch for player logging to my server without updating the 'realmlist.wtf' file manually

### DISCUSSION THREAD

**Discussion by wplatform** [azerothcore/azerothcore-wotlk#3963] [General]
I want to know how to release a patch for player logging to my server without updating the 'realmlist.wtf' file manually

**Comment by pangolp** [comment]
I developed a small launcher, which allows you to, among other things, modify the reamlist and delete the cache folder for example. It only works in Windows at the moment, but from azeroth, you want to work in a multiplatform launcher.

If you wish, contact me by discord, my name is Stevej and I will try to help you, although I warn you that my English is bad and I use translators to communicate many times.

**Comment by wplatform** [comment]
Sorry, the Discord seems not to work in China. It can't log in to the server even I have VPN network.

**Comment by BarbzYHOOL** [comment]
we had chinese people on our discord

**Comment by pangolp** [comment]
But they must be using a vpn maybe. You would have to ask on the China channel, as they do to access, but I don't really understand English, let alone Chinese.

**Comment by wplatform** [comment]
@pangolp  What is your native language and can you speak Chinese? I am a lot of problems regarding the game. I am just a Java developer, not c++, and I rarely have the knowledge about this game. I test the game for a while and found it just support the client with 12340 builds. use the highest client(13930) will crash.

**Comment by pangolp** [comment]
My native language is Spanish. And if it weren't for the translator, who works well inside everything even though he sometimes makes some mistakes in the translations, I wouldn't be able to do these publications. @wplatform

**Comment by BarbzYHOOL** [comment]
I answered on your OTHER issue for client 13930 @wplatform

**Comment by BarbzYHOOL** [comment]
here https://github.com/azerothcore/azerothcore-wotlk/issues/2564

**Comment by PkllonG** [comment]
把版本13930的wow.exe 替换为12340的wow.exe 就可以了。 13930的wow.exe 存在很多问题。

**Comment by PkllonG** [comment]
需要的话可以加我QQ：190532533

**Comment by PkllonG** [comment]
Discord 我很长时间都上不去了。。

**Comment by PkllonG** [comment]
Also, the 13930 wow. Exe in China is not 13930a, but 13930b

**Comment by wplatform** [comment]
@BarbzYHOOL  Sorry to reply to your message too late and Thank you for your kind answer to my question.  I want to know how to release a custom client that connects to my server. I want to .exe file, not the .bat. Thanks.
